README.md

```diff
@@ -1,10 +1,9 @@
 # Filecoin Devgrants
+> We are continously reviewing applictions for [open grants](https://github.com/filecoin-project/devgrants/blob/master/open-grant-proposals/open-proposal-template.md), [RFPS](https://github.com/filecoin-project/devgrants/tree/master/rfps), and [microgrants](https://github.com/filecoin-project/devgrants/blob/master/microgrants/microgrants.md). Submit your proposals now for priority consideration.
```

Comment:
Since this is at the top of the page, people reading it won't know the difference between each grant type or how to submit proposals yet.

Let's remove from here, and include a line about review cadence at the end of each section in "Types of Grants", e.g. "RFPs proposals are reviewed monthly. The deadline for the next wave is **July 30, 2021**." or "Microgrant applications are reviewed every 2 weeks." (maybe leave the date out of here so we're not constantly editing.

Reply:
Great!